backend/src/types.ts

```diff
@@ -555,6 +555,7 @@ export type TagContent = {
 
 export type ImageTagInfo = {
   name: string;
+  from_name: string;
```

Comment:
in testing, tag.from.name should always have a value when stemming from the initial imagestream defs https://github.com/opendatahub-io/odh-manifests/blob/master/notebook-images/base/jupyter-datascience-notebook-imagestream.yaml#L22